test/language/generators/generator.expression.no-name.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+    Generator can be declared with GeneratorExpression syntax
+es6id: 14.4
+author: Sam Mikes
+description: can create generator function expressions (no name)
+---*/
+
+var f = function *(a) { yield a+1; return; };
+
+assert.sameValue(f.name, '');
```

Comment:
Using an array works, alternatively you can use a comma expression `var a = (0, function(){})` or if you like to confuse readers `var a; (a) = function(){};`. :wink: